mixpanel_test.go

```diff
@@ -63,52 +65,85 @@ func TestTrack(t *testing.T) {
 	}
 }
 
-func TestPeopleOperations(t *testing.T) {
```

Comment:
Did you remove the People tests that were here? `TestPeopleOperations` and `TestPeopleTrack`

Reply:
As the first two commit say, TestPeopleTrack was a duplicate of the TestTrack method and TestPeopleOperations tests the Update method so it was renamed to reflect this.